docs/ides/configuring-emacs-tramp.md

```diff
@@ -0,0 +1,56 @@
+# Configuring Emacs TRAMP
+[Emacs TRAMP](https://www.emacswiki.org/emacs/TrampMode) is a method of running editing operations on a remote server.
+
+## Connecting To A Workspace
+To connect to your workspace first run:
+
+```
+coder config-ssh
+```
+
+Then you can connect to your workspace by its name in the format: `coder.<name>`.
+
+In Emacs type `C-x d` and then input: `/ssh:coder.<name>:` and hit enter. This will open up Dired on the workspace's home directory.
+
+## Directory Tracking
+### `ansi-term`
+If you run your terminal in Emacs via `ansi-term` then you might run into a problem where while SSH-ed into a workspace Emacs will not change its `default-directory` to open files in the directory your shell is in.
+
+To fix this:
+
+1. In your Emacs `init.el` file add:
+   ```lisp
+   (setq tramp-default-method "ssh")
+   ```
```

Comment:
This is not strictly necessary is it?  The default of `scp` worked for me.  Setting it to `ssh` could be useful if doing multi-hopping but to me this seems beyond the scope of connecting to Coder.

Reply:
I suppose it isn't strictly necessary. I had been accessing my coder workspace via the `/ssh:` TRAMP method, but SCP works pretty well too. If you prefer to access via `/ssh:` you need to set the default access method otherwise when you navigate around in `ansi-term` inferior shell and try to `C-f` open a file it will put `/scp:coder...`. 

Perhaps I could update the instructions to access via SCP and those who want to do more advanced stuff like SSH can figure this out on their own?

Reply:
I think you bring up a good point.  It may be non-obvious why `ansi-term` is using `scp` instead of `ssh`.  Maybe we can say something like "ansi-term will use `scp` as the default connection method; to change this see `tramp-default-method`" (no need for that exact wording though).